public/js/src/client.js

```diff
@@ -1,10 +1,21 @@
+/*
+	You're doing everything in the ready state.  
+	Is there anything that could be done asynchronously,
+	while the document is loading?  The initial $.get, 
+	for instance?  Could this retrieve data, then wait 
+	till DOM ready to insert it?
+*/
 $(document).ready(function(){
+	"use strict";
 
 	var OPERATIONS = {};
 
+	var $window = $(window);
```

Comment:
Okay!  What kind of downsides does the chaching result in?  For example, if the content of the window changes, will the jquery object reflect that change? I guess my broader question is, what kinds of operations demand a refresh in the jquery object, and which ones don't (or if you tell me the first, I can infer all others are in the latter category)?